src/GDGUkraine/lib/forms/registration_forms.py

```diff
@@ -0,0 +1,268 @@
+import logging
+
+from wtforms.form import Form
+from wtforms.fields import (
+    StringField, TextAreaField, SelectField,
+    BooleanField, SelectMultipleField,
+)
+from wtforms.widgets import ListWidget, RadioInput, CheckboxInput
+from wtforms.validators import Required, Length, Email, URL, Optional
+
+from GDGUkraine.model import (
+    EXPERIENCE_CHOICES, ENGLISH_CHOICES,
+    GENDER_CHOICES, TSHIRT_CHOICES,
+)
+
+from .widgets import InlineWidget
+
+
+logger = logging.getLogger(__name__)
+
+
+class RegistrationForm(Form):
+
+    name = StringField(
```

Comment:
1. Any **trusted** package like _WTForms from yaml_? 
2. Storing crucial validation in yaml config seems dangerous to me as it will make code more fragile.